spec/features/partner_assets_page_spec.rb

```diff
@@ -3,11 +3,7 @@
 RSpec.describe "Partner assets page" do
   let(:partner_assets_heading) { I18n.t("estimate_flow.partner_assets.assets.legend") }
 
-  around do |example|
-    Flipper.enable(:partner)
-    example.run
-    Flipper.disable(:partner)
-  end
+  enable_partner_screens
```

Comment:
There is a more idiomatic version of this in EL-472 - I think I will break it out into a separate PR

Reply:
Yes please, that looks great!